Makefile

```diff
@@ -21,13 +21,17 @@ olympus-docker:
 
 .PHONY: dev
 dev:
-	docker-compose -p athensdev up -d mysql
-	docker-compose -p athensdev up -d postgres
+	# docker-compose -p athensdev up -d mysql
+	# docker-compose -p athensdev up -d postgres
```

Comment:
@arschles ops, didn't meant this to be in the PR. But now that it's here: we're not using mysql/postgres in development are we? I keep commenting this out because they take a while to build up and tear down

Reply:
@marwan-at-work I think you need them up if you want to run i.e. the storage tests but you're right - not in development since the storage defaults to mem or fs i think.
Maybe we should have a separate target for tests?

Reply:
agree - we should have a separate target for test deps. is it cool if we leave these uncommented and do that separation in a follow-up PR?